tiles/src/main/java/com/protomaps/basemap/Basemap.java

```diff
@@ -112,6 +116,21 @@ public String attribution() {
       """.trim();
   }
 
+  @Override
+  public Map<String, String> extraArchiveMetadata() {
+    Map<String, String> result = new HashMap<>();
+
+    FontRegistry fontRegistry = FontRegistry.getInstance();
+    List<String> scripts = fontRegistry.getScripts();
+
+    for (String script : scripts) {
+      result.put("pgf:" + script.toLowerCase() + ":name", fontRegistry.getName(script));
```

Comment:
It wasn't clear to me that `pgf` stands for "positioned glyph font", consider adding a comment above this line to explain that?

Reply:
Good point. I was also wondering if we have a schema documentation somewhere?

Reply:
In addition to the inline comment in the code here, we should probably add a new "names.md" or `localization.md` file next to https://github.com/protomaps/docs/blob/main/basemaps/layers.md in the Protomaps docs repo?

Tilezen had [this](https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md#feature-names), which we should only lightly crib from. 

Natural Earth's [table of common languages](https://github.com/nvkelso/natural-earth-vector/blob/master/packages/Natural_Earth_quick_start/LOCALIZATION.md#setting-names-localization) is useful, too, and add a script column to that?

<img width="750" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/02c0ecd1-2189-4f95-9bfc-3ca989d60479">

...snip...

Reply:
Stubbed out in new docs PR: https://github.com/protomaps/docs/pull/43/files